Code/Yahoo.Yui.Compressor.Tests/JavaScriptCompressorTaskTests.cs

```diff
@@ -182,6 +182,23 @@ public void An_Error_Is_Not_Logged_If_The_Output_File_Has_The_Same_Name_But_A_Di
             Assert.That(BuildEngine.DoesNotContainError(compressor.BuildEngine, ("Output file cannot be the same as source file(s).")));
         }
 
+        [Test]
+        public void If_OutputFile_Directory_Does_Not_Exist_It_Is_Created()
+        {
+            // Arange
+            var compressor = CreateCompressorTask();
+            compressor.SourceFiles = new ITaskItem[] { new TaskItem("Javascript Files\\SampleJavaScript1.js") };
+            compressor.OutputFile = "NewOutputDir\\Output.js";
+            if (Directory.Exists(Path.GetFullPath("NewOutputDir")))
+                Directory.Delete(Path.GetFullPath("NewOutputDir"), true);

```

Comment:
Ok, I was thinking of File.Delete which can silently not delete a file (e.g. it is in use with delete sharing on), perhaps directory deletion doesn't suffer from the same problem.
Forgot to add, can you delete the directory afterwards also?  It's always good to clean up after a test (suggest a try/catch to ensure the delete always happens) rather than having directories linger afterwards...